### PR TITLE
First draft of new wiki documentation

### PR DESCRIPTION
**Pull request by jaxels10**
[Tech-879](https://team-1612171154669.atlassian.net/jira/software/c/projects/TECH/boards/8?modal=detail&selectedIssue=TECH-879)
This is the first draft of the new wiki documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,64 @@
-# Wiki 
-The wiki contains various documentation and guidelines for the projects developed in the distributed technologies team. 
+<img src="images/Energinet-logo.png" width="250" style="margin-bottom: 3%">
```
Comment (klausenbusk) [inline README.md:1]:
```suggestion
![Energinet logo](images/Energinet-logo.png)
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,89 @@
...
+
+These are just some of the many requirements there were for the platform and many more have come forward during development. 
+
+### Yocto
+After having completed a successfull PoC of the platform, we soon realized that the build method we were using was slow and did not have all the features we needed. We spent some time looking for other build tools and found the Yocto project. Yocto takes advantage of caching layers of the operating system, which means that if anything changes in a particular layer, only that layer has to be recompiled. This sped up OS compiling significantly and has allowed us to automatic builds. 
+
+# Decision log
+Any decisions made during meetings about SCCP that can be open to the public should be listed here.
+
+- 11.01.2022: A decision was made about looking into the possibility of packaging Yggdrasil and only requiring an environment repository to configure the platform.
```
Comment (klausenbusk) [inline README.md:64]: Perhaps this should be split out into its own file?

**Review by skovbakke — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (klausenbusk) [inline README.md:64]: https://xkcd.com/1179/

```suggestion
# Decision log
Any decisions made during meetings about SCCP that can be open to the public should be listed here.

- 2021.01.11: A decision was made about looking into the possibility of packaging Yggdrasil and only requiring an environment repository to configure the platform.
```

**Comment by skovbakke** [comment]
From my understanding, after talking to @jacobcrawford, we do not want TECH-828 to be covered by this PR. Instead we will create a separate PR for that.  The landing page should cover general information and not anything too specific.

File: README.md (hunk 2 above)
Comment (jaxels10) [reply README.md]: I have done this in the newest commit

**Review by skovbakke — DISMISSED** [review]
LGTM 😎

File: README.md [hunk 3]
```diff
@@ -1,8 +1,88 @@
-# Wiki 
-The wiki contains various documentation and guidelines for the projects developed in the distributed technologies team. 
+![Energinet logo](images/Energinet-logo.png)
+
+# Overview
+The Secure Compute and Communication Platform (SCCP) project is an attempt at creating a modern, fast, secure, robust, and uniform platform for running a variety of application suites for running and monitoring the Danish energy grid.
```
Comment (klausenbusk) [inline README.md:4]:
```suggestion
The Secure Compute and Communication Platform (SCCP) project is an attempt at creating a modern, fast, secure, robust, and uniform platform for running a variety of application suites for running and monitoring infrastructure.
```

File: README.md [hunk 4]
```diff
@@ -1,8 +1,88 @@
...
+The platform is made up of multiple components: 
+
+### Hardware
+Hardware plays an essential part in the design of the platform. The platform rests on multiple servers that participate in a kubernetes cluster which provide redundancy.
```
Comment (klausenbusk) [inline README.md:8]:
```suggestion
Hardware plays an essential part in the design of the platform. The platform rests on multiple servers that participate in a Kubernetes cluster which provide redundancy. 
```

**Review by klausenbusk — APPROVED** [review]
LGTM 🎉

**Review by klausenbusk — APPROVED** [review]
LGTM 🎉

**Review by jacobcrawford — APPROVED** [review]
LGTA (Andreas)

**Pull request merged**